Week8-2/Extensions/UIImageView+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UIImageView+Extensions.swift
+//  Week8-2
+//
+//  Created by Mehmet Salih ÇELİK on 19.11.2021.
+//
+
+import Foundation
+import Kingfisher
+
+extension UIImageView {
+    func configureKF(url: String) {
```

Comment:
optional string alacak şekilde düzenlenebilir. Her seferinde unwrap yapmamak için

Reply:
fixed